packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -22,7 +21,6 @@ export const NetworkSelectionContainer = ({
   children: React.ReactNode
 }) => {
   const { chain } = useNetwork()
-  const { switchNetwork } = useSwitchNetworkWithConfig()
```

Comment:
why is this changed? fairly sure we want to use `useSwitchNetworkWithConfig` instead of `switchNetwork` here